lib/call-session.js

```diff
@@ -864,6 +864,12 @@ Duration=${payload.duration} `
         if (!offerMedia.flags.includes('asymmetric')) offerMedia.flags.push('asymmetric');
         offerMedia.flags = offerMedia.flags.filter((f) => f !== 'media handover');
       }
+      const acceptCodecs = process.env.JAMBONES_ACCEPT_AND_TRANSCODE ?
+        process.env.JAMBONES_ACCEPT_AND_TRANSCODE :
+        process.env.JAMBONES_ACCEPT_G729 ? 'g729' : '';
+      offerMedia = { ...offerMedia, ...(acceptCodecs && { codec: { mask: acceptCodecs, transcode: 'pcmu,pcma' } })};
```

Comment:
Do we need to check the direction of the re-invite so that we don't override wrong thing? could you please help to check this?

If feature server send RE-INVITE with opus first, then with this change the callee as webrtc will receive pcmu and pcma